data/theme/gnome-shell.css

```diff
@@ -1883,11 +1883,29 @@ stage {
 }
 
 .discovery-feed-button {
-  width: 44px;
-  height: 23px;
-  background-image: url(resource:///org/gnome/shell/theme/discovery-feed-open-tab.png); }
+    width: 44px;
+    height: 22px;
+    border-radius: 0px 0px 22px 22px;
+
+    background-gradient-start: rgba(255,95,42,0.9);
+    background-gradient-end: rgba(248,63,0,0.9);
+    background-gradient-direction: vertical;
+
+    box-shadow: inset 0 -1px 1px rgba(0,0,0,0.25);
+
+    color: #eeeeec;
+}
+
 .discovery-feed-button:hover {
-  background-image: url(resource:///org/gnome/shell/theme/discovery-feed-open-tab_hover.png); }
+    background-gradient-start: rgba(255,95,42,1.0);
+    background-gradient-end: rgba(248,63,0,1.0);
+
+    color: white;
```

Comment:
It's supposed to be white only on hover. See also the "search icon" theming further down in the file.